web/src/components/utils/PopulateForms/components/CheckBox.vue

```diff
@@ -1,60 +1,61 @@
 <template>
-    <div :class="checkbox" >
-        <div 
-            :style="{position: 'absolute',
-            margin:'0 0 0 '+ boxMargin,
+    <div :class="checkbox">
+        <div :style="{
+            position: 'absolute',
+            margin: '0 0 0 ' + boxMargin,
             display: inline,
-            border: '1px solid #000!important',             
-            width: '1em', 
-            transform:'translate('+shift+'px,4px)',           
-            height: '1em',           
-            fontSize: '1.0em'}">
-            <div :style="{transform:'translate('+shiftmark+'px,-4px)'}" v-html="getCheck()"/>
-        </div> 
-        <div :style="{display:'inline-block', marginLeft: marginLeft, textIndent: 0 }" v-html="text">{{text}}</div>
+            border: '1px solid #000!important',
+            width: '1em',
+            transform: 'translate(' + shift + 'px,4px)',
+            height: '1em',
+            backgroundColor: '#dedede',
```

Comment:
This will change the default background checkbox color for every single form using this component. Is this intended? There seem to be over 100 imports of this component across the project

Reply:
Yes Josh this is intentional.  I think all the new forms need this background color for the check box from the feedback from the testers, So i added that to the component that is shared which is used on all the forms.